docs/getting-started.md

```diff
@@ -0,0 +1,96 @@
+
+## 1. Verify that the minimum hardware and network requirements are met. 
+
+Harvester is built for bare metal servers using enterprise-grade open-source software components. The installer automatically checks the hardware and displays warning messages if the minimum [requirements](../install/requirements) are not met. 
+
+## 2. Prepare the installation files based on the installation method that you want to use. 
+
+You can download the installation files from the [Harvester Releases](https://github.com/harvester/harvester/releases) page. The **Downloads** section of the release notes contains links to the ISO files and related artifacts. The following types of ISO files are available: 
+
+- **Full ISO**: Contains the core operating system components and all required container images, which are preloaded during installation. You must use a full ISO when installing Harvester behind a firewall or proxy, and in environments without internet connectivity. 
+- [**Net install ISO**](../install/net-install): Contains only the core operating system components. After installation is completed, the operating system pulls all required container images from the internet (mostly from Docker Hub). 
+
+
+| Method | Required Installation Files | Other Requirements |
+| --- | --- | --- |
+| [ISO](../install/index) | ISO | N/A |
+| [USB](../install/usb-install) | ISO | USB flash drive; utility such as [balenaEtcher](https://etcher.balena.io/) or the Linux [dd command](https://man7.org/linux/man-pages/man1/dd.1.html) |
+| [PXE](../install/pxe-boot-install) | ISO, Linux kernel image (vmlinuz), initrd, SquashFS image | Directory on the HTTP server (for serving boot files); iPXE boot scripts (for automatic installation); DHCP server configuration |
+
+## 3. Prepare the cluster configuration requirements. 
+
+- Cluster token: ASCII string that nodes use when joining the cluster 
+- Fixed IP address for each node: May be assigned statically or using DHCP (host reservation) 
+- Fixed virtual IP address (VIP) to be used as the cluster management address: VIP that nodes point to when joining the cluster and that you connect to when performing administration tasks after the cluster is deployed 
+- Addresses of DNS servers, NTP servers, and the proxy server (if necessary) 
+
+## 4. Deploy the first cluster node. 
+
+Deployment involves installing the operating system and other components on the host, and then rebooting once installation is completed. Deploying the first node creates the cluster, and the first node is assigned the management node by default. 
+    
+During installation, you must configure node settings, define the cluster management address (VIP) and the cluster token, and specify other information. If necessary, you can configure more settings related to installation, the operating system, and advanced features using a [Harvester configuration](../install/harvester-configuration/) file. 
```

Comment:
Don’t really understand what’s the settings about ‘the operating system’ here. Do u mean ‘sysctl’ related settings?

Reply:
We remove the descriptions of the settings if you prefer that.

_If necessary, you can configure more settings using a [Harvester configuration](https://667c032666edd0c923153f3d--harvester-preview.netlify.app/install/harvester-configuration/) file._